ui/src/routes.js

```diff
@@ -1,7 +1,18 @@
 import React from 'react'
 import { Route } from 'react-router-dom'
 import Personal from './personal/Personal'
+import ArtistView from './common/ArtistDetail'
 
-const routes = [<Route exact path="/personal" render={() => <Personal />} />]
+const routes = [
+  <Route exact path="/personal" render={() => <Personal />} />,
+  <Route
```

Comment:
I'll do a more throughly review later, but you don't need to add a new route. Just add the new ArtistView as the Show page for artist, similar to what we do with AlbumShow for album: https://github.com/navidrome/navidrome/blob/b441260186b3b88069fcb4d8c84ef06e1a6a4dbb/ui/src/album/index.js#L4-L7

Also, renaming `ArtistView` to `ArtistShow` would be better to keep consistency

Reply:
Thanks for the review, deluan; I have pushed the required changes.